helm/istio/charts/sidecarInjectorWebhook/templates/mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: istio-sidecar-injector-{{ .Release.Namespace }}
+  labels:
+    app: {{ template "sidecar-injector.name" . }}
+    chart: {{ template "sidecar-injector.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  annotations:
+    "helm.sh/resource-policy": keep
+webhooks:
+  - name: sidecar-injector.istio.io
+    clientConfig:
+      service:
+        name: istio-sidecar-injector
+        namespace: {{ .Release.Namespace }}
+        path: "/inject"
+      caBundle: ""
+    rules:
+      - operations: [ "CREATE" ]
+        apiGroups: [""]
+        apiVersions: ["v1"]
+        resources: ["pods"]
+    failurePolicy: Fail
+    namespaceSelector:
+      matchExpressions:
+      - key: maistra.io/member-of
+        operator: In
+        values:
+        - {{ .Release.Namespace }}
+      - key: maistra.io/ignore-namespace
+        operator: DoesNotExist
+      - key: istio.openshift.com/ignore-namespace
+        operator: DoesNotExist
+      matchExpressions:
+      - key: maistra.io/ignore-namespace
+        operator: DoesNotExist
+      - key: istio.openshift.com/ignore-namespace
+        operator: DoesNotExist
```

Comment:
These 5 lines are duplicated.

Reply:
Also, I think we must get rid of `istio.openshift.com` stuff? @rcernich